### ISSUE TITLE
CTRL+F4 is not closing current tab

### ISSUE THREAD

**Issue by TheMagnificent11** [microsoft/terminal#14709]
### Windows Terminal version

1.15.3465.0

### Windows build number

10.0.19044.0

### Steps to reproduce

1. Open Windows Terminal
2. Press CTRL+Shift+1 to open a new tab
3. Types something into the new shell (to confirm that it has focus)
4. Press CTRL+F4

### Expected Behavior

The current tab is closed

### Actual Behavior

Nothing

**Comment by zadjii-msft** [comment]
Did you bind `closeTab` or `closePane` to `ctrl+f4`? 

Did you try [searching for duplicates](https://github.com/microsoft/terminal/issues?q=is%3Aissue+ctrl%2Bf4)?

**Comment by TheMagnificent11** [comment]
I did try searching for duplicates, but didn't find any that were open, and the closed ones said that the issue was fixed.

It turns out that `closePane` was mapped to `CTRL+Shift+w` and changing it to `CTRL+F4` fixed my issue.

However, before changing the action binding, hovering over the "x" button for a tab said that `CTRL+F4` was the shortcut, not `CTRL+Shift+w`.

So, I guess the bug is that hovering over the close button gives the wrong shortcut.
